### PR TITLE
Fix small issues

### LINKED ISSUE #1264: Create constants for the APY filenames
Create constants for the APY filenames in order to make them less error-prone.

_Originally posted by @Karrenbelt in https://github.com/valory-xyz/open-autonomy/pull/1254#discussion_r960017879_

### PR DESCRIPTION
**Pull request by Adamantios**
## Proposed changes

Addresses various small issues.

## Fixes

Fixes #1264.

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [x] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [x] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

n/a

### CODE REVIEW

File: packages/valory/skills/apy_estimation_abci/models.py [hunk 1]
```diff
@@ -200,11 +200,13 @@ def _validate_parsed_hacky_pair_ids(pair_ids: PairIdsType) -> None:
...
-        for dex_id in dex_ids:
-            if not isinstance(dex_id, str):
-                _raise_incorrect_hacky_initialization()
+        if (
+            isinstance(dex_name, str)
+            and isinstance(dex_ids, list)
+            and all(isinstance(_id, str) for _id in dex_ids)
+        ):
+            continue
+        _raise_incorrect_hacky_initialization()
```
Comment (Adamantios) [inline packages/valory/skills/apy_estimation_abci/models.py:209]: Refactors based on https://github.com/valory-xyz/open-autonomy/pull/1171#discussion_r950279959.

File: .github/workflows/main_workflow.yml [hunk 2]
```diff
@@ -391,7 +391,7 @@ jobs:
         os: [ ubuntu-latest ]
         python-version: [ "3.10" ]
 
-    timeout-minutes: 140
+    timeout-minutes: 150
```
Comment (Adamantios) [inline .github/workflows/main_workflow.yml:394]: Increased e2e tests' timeout to give room for the increased APY round timeout.

File: tests/test_agents/test_apy_estimation_abci.py [hunk 3]
```diff
@@ -54,19 +54,14 @@ class BaseTestABCIAPYEstimationSkillNormalExecution(BaseTestEnd2EndExecution):
     agent_package = "valory/apy_estimation_chained:0.1.0"
     skill_package = "valory/apy_estimation_chained_abci:0.1.0"
     happy_path = HAPPY_PATH
-    ROUND_TIMEOUT_SECONDS = 120
-    wait_to_finish = 240
+    ROUND_TIMEOUT_SECONDS = 480
+    wait_to_finish = 480
```
Comment (Adamantios) [inline tests/test_agents/test_apy_estimation_abci.py:58]: Increased round timeout to try to resolve APY 4 agents' e2e flakiness caused by the optimization not finishing.

File: tests/test_agents/test_apy_estimation_abci.py [hunk 4]
```diff
@@ -54,19 +54,14 @@ class BaseTestABCIAPYEstimationSkillNormalExecution(BaseTestEnd2EndExecution):
...
     extra_configs = [
         {
             "dotted_path": f"{__args_prefix}.ipfs_domain_name",
             "value": "/dns/localhost/tcp/5001/http",
         },
-        {
-            "dotted_path": f"{__args_prefix}.round_timeout_seconds",
-            "value": 300,
-            "type_": "float",
-        },
```
Comment (Adamantios) [inline tests/test_agents/test_apy_estimation_abci.py:69]: Was not necessary, because we have `ROUND_TIMEOUT_SECONDS` above.

**Review by Karrenbelt — APPROVED** [review]

**Review by Karrenbelt — APPROVED** [review]

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**